### ISSUE TITLE
[BUG] Software Updates package list not always rendered

### ISSUE THREAD

**Issue by moorereason** [marticliment/UniGetUI#2737]
### Please confirm these before moving forward

- [X] I have searched for my issue and have not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions.
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose)).

### UniGetUI Version

3.1.2-beta1

### Windows version, edition, and architecture

Windows 11 Education 10.0.22631 Build 22631

### Describe your issue

Although the **Software Updates** icon in the toolbar lists a number for updates available, the **Software Updates** page doesn't always render a list of packages.  The page renders as if no packages need to be updated.

The only workaround I've found is to resize the package list pane by moving the vertical divider between it and the the filter options.

I believe this problem was previously reported in #2627 which as closed after #2677 was merged and released in 3.1.2-beta0.  I originally saw this issue in 3.1.1 and was hoping #2677 would fix it, but the issue has occurred twice since updating to 3.1.2-beta1.

### Steps to reproduce the issue

The problem is intermittent, and I don't know how to its force reproduction.  But when it occurs, this is generally what happened:

1. Click the UniGetUI icon in the system tray.
2. Notice if the **Software Updates** icon in the taskbar shows available updates.
3. If not on the **Software Updates** page, click the **Software Updates** icon in the toolbar.
4. If the bug is triggered, the **Software Updates** page will render as if no updates are available (the package listing will be empty).
5. Do the workaround above, and the package listing shows all available updates as expected.

### UniGetUI Log

```text
[9/12/2024 8:19:29 AM]    __  __      _ ______     __  __  ______
                         / / / /___  (_) ____/__  / /_/ / / /  _/
                        / / / / __ \/ / / __/ _ \/ __/ / / // /
                       / /_/ / / / / / /_/ /  __/ /_/ /_/ // /
                       \____/_/ /_/_/\____/\___/\__/\____/___/
                           Welcome to UniGetUI Version 3.1.2-beta1
[9/12/2024 8:19:29 AM]
[9/12/2024 8:19:29 AM] Version Code:  3.1191
[9/12/2024 8:19:29 AM] Encoding Code Page set to 437
[9/12/2024 8:19:30 AM] Using bundled GSudo at C:\Program Files\WingetUI\Assets\Utilities\gsudo.exe
[9/12/2024 8:19:33 AM] Loaded language locale: en
[9/12/2024 8:19:35 AM] Randomly-generated background API auth token for the current session: nuli6g29wut9qlfvt09avhrjcbq8fyhe0vg28rpganfz659imhfswf6y9dz877sy
[9/12/2024 8:19:36 AM] Api running on http://localhost:7058
[9/12/2024 8:19:39 AM] Lang files were updated successfully from GitHub
[9/12/2024 8:19:39 AM] UniGetUI is up to date
[9/12/2024 8:19:40 AM] Command npm was not found on the system
[9/12/2024 8:19:40 AM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                       █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                       █ Name: Npm
                       █ Enabled: True
                       █ Found: False
                       █ THE MANAGER WAS NOT FOUND. PERHAPS IT IS NOT 
                       █ INSTALLED OR IT HAS BEEN MISCONFIGURED 
                       ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[9/12/2024 8:19:41 AM] Downloaded new icons and screenshots successfully!
[9/12/2024 8:19:41 AM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                       █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                       █ Name: .NET Tool
                       █ Enabled: True
                       █ Found: False
                       █ THE MANAGER WAS NOT FOUND. PERHAPS IT IS NOT 
                       █ INSTALLED OR IT HAS BEEN MISCONFIGURED 
                       ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[9/12/2024 8:19:41 AM] Command scoop was not found on the system
[9/12/2024 8:19:41 AM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                       █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                       █ Name: Scoop
                       █ Enabled: True
                       █ Found: False
                       █ THE MANAGER WAS NOT FOUND. PERHAPS IT IS NOT 
                       █ INSTALLED OR IT HAS BEEN MISCONFIGURED 
                       ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[9/12/2024 8:19:42 AM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                       █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                       █ Name: Chocolatey
                       █ Enabled: True
                       █ Found: True
                       █ Fancy exe name: choco.exe
                       █ Executable path: C:\Users\user\AppData\Local\UniGetUI\Chocolatey\choco.exe
                       █ Call arguments: 
                       █ Version: 
                       █   2.2.2
                       ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[9/12/2024 8:19:45 AM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                       █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                       █ Name: Winget
                       █ Enabled: True
                       █ Found: True
                       █ Fancy exe name: winget.exe
                       █ Executable path: C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe
                       █ Call arguments: 
                       █ Version: 
                       █   System WinGet CLI Version: v1.8.1911
                       █   Microsoft.WinGet.Client PSModule version: ""
                       █   Using Native WinGet helper (COM Api)
                       ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[9/12/2024 8:19:45 AM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                       █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                       █ Name: Pip
                       █ Enabled: True
                       █ Found: True
                       █ Fancy exe name: pip
                       █ Executable path: C:\Users\user\AppData\Local\Microsoft\WindowsApps\python.exe
                       █ Call arguments:  -m pip
                       █ Version: 
                       █   pip 24.2 from C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.1776.0_x64__qbz5n2kfra8p0\Lib\site-packages\pip (python 3.12)
                       ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[9/12/2024 8:19:51 AM] PowerShell sources took too long to load, using known sources as default
[9/12/2024 8:19:51 AM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                       █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                       █ Name: PowerShell
                       █ Enabled: True
                       █ Found: True
                       █ Fancy exe name: powershell.exe
                       █ Executable path: C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe
                       █ Call arguments:  -NoProfile -Command
                       █ Version: 
                       █   Name                           Value                                                                                   
                       █   ----                           -----                                                                                   
                       █   PSVersion                      5.1.22621.3958                                                                          
                       █   PSEdition                      Desktop                                                                                 
                       █   PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                 
                       █   BuildVersion                   10.0.22621.3958                                                                         
                       █   CLRVersion                     4.0.30319.42000                                                                         
                       █   WSManStackVersion              3.0                                                                                     
                       █   PSRemotingProtocolVersion      2.3                                                                                     
                       █   SerializationVersion           1.1.0.1
                       ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[9/12/2024 8:19:52 AM] PowerShell7 sources took too long to load, using known sources as default
[9/12/2024 8:19:52 AM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                       █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                       █ Name: PowerShell7
                       █ Enabled: True
                       █ Found: True
                       █ Fancy exe name: pwsh.exe
                       █ Executable path: C:\Program Files\PowerShell\7\pwsh.exe
                       █ Call arguments:  -NoProfile -Command
                       █ Version: 
                       █   PowerShell 7.4.5
                       ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[9/12/2024 8:19:53 AM] LoadComponentsAsync finished executing. All managers loaded. Proceeding to interface.
[9/12/2024 8:19:57 AM] Microsoft.Powershell.Client detection has been forcefully skipped as the module is not required on the current context
[9/12/2024 8:19:57 AM] Dependency WinGet PowerShell Module for manager Winget is present
[9/12/2024 8:20:05 AM] Found 1 installed packages from Pip
[9/12/2024 8:20:05 AM] Found 12 installed packages from Chocolatey
[9/12/2024 8:20:10 AM] Found 0 available updates from Chocolatey
[9/12/2024 8:20:14 AM] Found 0 available updates from Pip
[9/12/2024 8:20:24 AM] Found 0 installed packages from PowerShell7
[9/12/2024 8:20:30 AM] Found 1 installed packages from PowerShell
[9/12/2024 8:20:32 AM] Found 4 available updates from Winget
[9/12/2024 8:20:34 AM] Found 179 installed packages from Winget
[9/12/2024 8:20:45 AM] Found 0 available updates from PowerShell7
[9/12/2024 8:20:53 AM] Error finding updates on manager PowerShell
[9/12/2024 8:20:53 AM] System.TimeoutException: The operation has timed out.
                          at UniGetUI.PackageEngine.ManagerClasses.Manager.PackageManager.GetAvailableUpdates() in C:\SomePrograms\WingetUI-Store\src\UniGetUI.PackageEngine.PackageManagerClasses\Manager\PackageManager.cs:line 194
[9/12/2024 8:20:58 AM] Found 1 installed packages from Pip
[9/12/2024 8:21:00 AM] Found 12 installed packages from Chocolatey
[9/12/2024 8:21:02 AM] Found 179 installed packages from Winget
[9/12/2024 8:21:03 AM] Found 0 installed packages from PowerShell7
[9/12/2024 8:21:05 AM] Found 1 installed packages from PowerShell
[9/12/2024 9:02:45 AM] Non-MSStore WinGet Native Icons have been forcefully disabled on code
[9/12/2024 9:02:45 AM] Loaded icon with URL=UniGetUI.Core.IconEngine.CacheableIcon for package Id=Amazon.AWSCLI
[9/12/2024 9:02:45 AM] Non-MSStore WinGet Native Icons have been forcefully disabled on code
[9/12/2024 9:02:45 AM] Loaded icon with URL=UniGetUI.Core.IconEngine.CacheableIcon for package Id=Postman.Postman
[9/12/2024 9:02:45 AM] Non-MSStore WinGet Native Icons have been forcefully disabled on code
[9/12/2024 9:02:45 AM] Loaded icon with URL=UniGetUI.Core.IconEngine.CacheableIcon for package Id=VideoLAN.VLC
[9/12/2024 9:02:45 AM] Icon for package VideoLAN.VLC stored on C:\Users\user\AppData\Local\UniGetUI\CachedMedia\Winget\VideoLAN.VLC.png
[9/12/2024 9:02:45 AM] Loaded operation parameters for package id=Amazon.AWSCLI on manager Winget and operation Update: update --id "Amazon.AWSCLI" --exact --source winget --accept-source-agreements --disable-interactivity --silent --include-unknown --accept-package-agreements --force
[9/12/2024 9:02:45 AM] Icon for package Amazon.AWSCLI stored on C:\Users\user\AppData\Local\UniGetUI\CachedMedia\Winget\Amazon.AWSCLI.png
[9/12/2024 9:02:45 AM] Icon for package Postman.Postman stored on C:\Users\user\AppData\Local\UniGetUI\CachedMedia\Winget\Postman.Postman.png
[9/12/2024 9:06:15 AM] Loaded operation parameters for package id=Postman.Postman on manager Winget and operation Update: update --id "Postman.Postman" --exact --source winget --accept-source-agreements --disable-interactivity --silent --include-unknown --accept-package-agreements --force
[9/12/2024 9:07:13 AM] Loaded operation parameters for package id=VideoLAN.VLC on manager Winget and operation Update: update --id "VideoLAN.VLC" --exact --source winget --accept-source-agreements --disable-interactivity --silent --include-unknown --accept-package-agreements --force
[9/12/2024 9:19:40 AM] UniGetUI is up to date
[9/12/2024 9:20:57 AM] Found 0 available updates from Pip
[9/12/2024 9:21:01 AM] Found 0 available updates from Chocolatey
[9/12/2024 9:21:02 AM] Found 0 available updates from PowerShell
[9/12/2024 9:21:02 AM] Found 2 available updates from Winget
[9/12/2024 9:21:03 AM] Found 0 available updates from PowerShell7
[9/12/2024 10:26:38 AM] UniGetUI is up to date
[9/12/2024 10:28:05 AM] Found 0 available updates from Pip
[9/12/2024 10:28:06 AM] Found 0 available updates from Chocolatey
[9/12/2024 10:28:09 AM] Found 0 available updates from PowerShell
[9/12/2024 10:28:10 AM] Found 0 available updates from PowerShell7
[9/12/2024 10:28:10 AM] Found 2 available updates from Winget
[9/12/2024 11:26:38 AM] UniGetUI is up to date
[9/12/2024 11:28:13 AM] Found 0 available updates from Pip
[9/12/2024 11:28:16 AM] Found 0 available updates from Chocolatey
[9/12/2024 11:28:19 AM] Found 0 available updates from PowerShell
[9/12/2024 11:28:20 AM] Found 2 available updates from Winget
[9/12/2024 11:28:20 AM] Found 0 available updates from PowerShell7
[9/12/2024 12:26:38 PM] UniGetUI is up to date
[9/12/2024 12:28:26 PM] Found 0 available updates from Pip
[9/12/2024 12:28:30 PM] Found 0 available updates from Chocolatey
[9/12/2024 12:28:34 PM] Found 2 available updates from Winget
[9/12/2024 12:28:35 PM] Found 0 available updates from PowerShell7
[9/12/2024 12:28:35 PM] Found 0 available updates from PowerShell
[9/12/2024 1:26:39 PM] UniGetUI is up to date
[9/12/2024 1:28:40 PM] Found 0 available updates from Pip
[9/12/2024 1:28:42 PM] Found 0 available updates from Chocolatey
[9/12/2024 1:28:47 PM] Found 2 available updates from Winget
[9/12/2024 1:28:49 PM] Found 0 available updates from PowerShell
[9/12/2024 1:28:49 PM] Found 0 available updates from PowerShell7
[9/12/2024 2:26:39 PM] UniGetUI is up to date
[9/12/2024 2:28:58 PM] Found 0 available updates from Pip
[9/12/2024 2:29:00 PM] Found 0 available updates from Chocolatey
[9/12/2024 2:29:02 PM] Found 2 available updates from Winget
[9/12/2024 2:29:06 PM] Found 0 available updates from PowerShell
[9/12/2024 2:29:09 PM] Found 0 available updates from PowerShell7
[9/12/2024 3:26:40 PM] UniGetUI is up to date
[9/12/2024 3:29:13 PM] Found 0 available updates from Pip
[9/12/2024 3:29:15 PM] Found 0 available updates from Chocolatey
[9/12/2024 3:29:18 PM] Found 2 available updates from Winget
[9/12/2024 3:29:21 PM] Found 0 available updates from PowerShell
[9/12/2024 3:29:23 PM] Found 0 available updates from PowerShell7
[9/12/2024 4:26:40 PM] UniGetUI is up to date
[9/12/2024 4:29:32 PM] Found 0 available updates from Chocolatey
[9/12/2024 4:29:33 PM] Found 0 available updates from Pip
[9/12/2024 4:29:33 PM] Found 2 available updates from Winget
[9/12/2024 4:29:39 PM] Found 0 available updates from PowerShell
[9/12/2024 4:29:40 PM] Found 0 available updates from PowerShell7
[9/12/2024 4:29:50 PM] Found 1 installed packages from Pip
[9/12/2024 4:29:51 PM] Found 12 installed packages from Chocolatey
[9/12/2024 4:29:52 PM] Found 1 installed packages from PowerShell
[9/12/2024 4:29:52 PM] Found 0 installed packages from PowerShell7
[9/12/2024 4:29:53 PM] Found 179 installed packages from Winget
[9/12/2024 4:29:53 PM] Adding missing package Amazon.AWSCLI to installed packages list
[9/12/2024 4:29:53 PM] Adding missing package Postman.Postman to installed packages list
[9/12/2024 4:29:53 PM] Adding missing package Microsoft.VisualStudioCode to installed packages list
[9/12/2024 4:29:53 PM] Adding missing package MSIX\Microsoft.Winget.Source_2024.912.1756.21_neutral__8wekyb3d8bbwe to installed packages list
[9/12/2024 4:29:54 PM] Non-MSStore WinGet Native Icons have been forcefully disabled on code
[9/12/2024 4:29:54 PM] Loaded icon with URL=UniGetUI.Core.IconEngine.CacheableIcon for package Id=VideoLAN.VLC
[9/12/2024 4:29:55 PM] Icon for package VideoLAN.VLC stored on C:\Users\user\AppData\Local\UniGetUI\CachedMedia\Winget\VideoLAN.VLC.png
[9/12/2024 4:29:56 PM] Loaded operation parameters for package id=VideoLAN.VLC on manager Winget and operation Update: update --id "VideoLAN.VLC" --exact --source winget --accept-source-agreements --disable-interactivity --silent --include-unknown --accept-package-agreements --force
[9/12/2024 6:42:49 PM] UniGetUI is up to date
[9/12/2024 6:45:55 PM] Found 0 available updates from Pip
[9/12/2024 6:45:56 PM] Found 0 available updates from Chocolatey
[9/12/2024 6:45:58 PM] Found 0 available updates from PowerShell
[9/12/2024 6:45:58 PM] Found 2 available updates from Winget
[9/12/2024 6:45:59 PM] Found 0 available updates from PowerShell7
[9/12/2024 7:42:49 PM] UniGetUI is up to date
[9/12/2024 7:46:07 PM] Found 0 available updates from Pip
[9/12/2024 7:46:08 PM] Found 0 available updates from Chocolatey
[9/12/2024 7:46:11 PM] Found 0 available updates from PowerShell
[9/12/2024 7:46:12 PM] Found 2 available updates from Winget
[9/12/2024 7:46:13 PM] Found 0 available updates from PowerShell7
[9/12/2024 8:42:49 PM] UniGetUI is up to date
[9/12/2024 8:46:16 PM] Found 0 available updates from Pip
[9/12/2024 8:46:17 PM] Found 0 available updates from Chocolatey
[9/12/2024 8:46:20 PM] Found 0 available updates from PowerShell
[9/12/2024 8:46:20 PM] Found 0 available updates from PowerShell7
[9/12/2024 8:46:21 PM] Found 2 available updates from Winget
[9/12/2024 9:42:49 PM] UniGetUI is up to date
[9/12/2024 9:46:24 PM] Found 0 available updates from Pip
[9/12/2024 9:46:25 PM] Found 0 available updates from Chocolatey
[9/12/2024 9:46:28 PM] Found 2 available updates from Winget
[9/12/2024 9:46:28 PM] Found 0 available updates from PowerShell
[9/12/2024 9:46:29 PM] Found 0 available updates from PowerShell7
[9/12/2024 10:42:50 PM] UniGetUI is up to date
[9/12/2024 10:46:34 PM] Found 0 available updates from Pip
[9/12/2024 10:46:36 PM] Found 0 available updates from Chocolatey
[9/12/2024 10:46:38 PM] Found 0 available updates from PowerShell
[9/12/2024 10:46:38 PM] Found 0 available updates from PowerShell7
[9/12/2024 10:46:38 PM] Found 3 available updates from Winget
[9/12/2024 11:42:50 PM] UniGetUI is up to date
[9/12/2024 11:46:43 PM] Found 0 available updates from Pip
[9/12/2024 11:46:47 PM] Found 0 available updates from Chocolatey
[9/12/2024 11:46:48 PM] Found 0 available updates from PowerShell
[9/12/2024 11:46:49 PM] Found 3 available updates from Winget
[9/12/2024 11:46:49 PM] Found 0 available updates from PowerShell7
[9/13/2024 12:42:50 AM] UniGetUI is up to date
[9/13/2024 12:46:52 AM] Found 0 available updates from Pip
[9/13/2024 12:46:56 AM] Found 0 available updates from Chocolatey
[9/13/2024 12:46:58 AM] Found 0 available updates from PowerShell
[9/13/2024 12:46:58 AM] Found 5 available updates from Winget
[9/13/2024 12:46:59 AM] Found 0 available updates from PowerShell7
[9/13/2024 1:42:51 AM] UniGetUI is up to date
[9/13/2024 1:47:03 AM] Found 0 available updates from Pip
[9/13/2024 1:47:06 AM] Found 0 available updates from Chocolatey
[9/13/2024 1:47:09 AM] Found 5 available updates from Winget
[9/13/2024 1:47:09 AM] Found 0 available updates from PowerShell
[9/13/2024 1:47:11 AM] Found 0 available updates from PowerShell7
[9/13/2024 2:42:51 AM] UniGetUI is up to date
[9/13/2024 2:47:17 AM] Found 0 available updates from Pip
[9/13/2024 2:47:19 AM] Found 0 available updates from Chocolatey
[9/13/2024 2:47:21 AM] Found 5 available updates from Winget
[9/13/2024 2:47:21 AM] Found 0 available updates from PowerShell
[9/13/2024 2:47:22 AM] Found 0 available updates from PowerShell7
[9/13/2024 3:42:51 AM] UniGetUI is up to date
[9/13/2024 3:47:30 AM] Found 0 available updates from Pip
[9/13/2024 3:47:31 AM] Found 0 available updates from Chocolatey
[9/13/2024 3:47:33 AM] Found 0 available updates from PowerShell
[9/13/2024 3:47:34 AM] Found 5 available updates from Winget
[9/13/2024 3:47:34 AM] Found 0 available updates from PowerShell7
[9/13/2024 4:42:51 AM] UniGetUI is up to date
[9/13/2024 4:47:45 AM] Found 0 available updates from Pip
[9/13/2024 4:47:45 AM] Found 0 available updates from Chocolatey
[9/13/2024 4:47:48 AM] Found 5 available updates from Winget
[9/13/2024 4:47:48 AM] Found 0 available updates from PowerShell
[9/13/2024 4:47:50 AM] Found 0 available updates from PowerShell7
[9/13/2024 5:42:52 AM] UniGetUI is up to date
[9/13/2024 5:47:53 AM] Found 0 available updates from Pip
[9/13/2024 5:47:56 AM] Found 0 available updates from Chocolatey
[9/13/2024 5:47:58 AM] Found 0 available updates from PowerShell7
[9/13/2024 5:47:59 AM] Found 5 available updates from Winget
[9/13/2024 5:47:59 AM] Found 0 available updates from PowerShell
[9/13/2024 6:42:52 AM] UniGetUI is up to date
[9/13/2024 6:48:03 AM] Found 0 available updates from Pip
[9/13/2024 6:48:06 AM] Found 0 available updates from Chocolatey
[9/13/2024 6:48:09 AM] Found 0 available updates from PowerShell
[9/13/2024 6:48:09 AM] Found 5 available updates from Winget
[9/13/2024 6:48:09 AM] Found 0 available updates from PowerShell7
[9/13/2024 7:42:52 AM] UniGetUI is up to date
[9/13/2024 8:20:49 AM] Found 0 available updates from Chocolatey
[9/13/2024 8:20:55 AM] Found 0 available updates from Pip
[9/13/2024 8:20:59 AM] Found 6 available updates from Winget
[9/13/2024 8:21:02 AM] Found 0 available updates from PowerShell7
[9/13/2024 8:21:02 AM] Found 0 available updates from PowerShell
[9/13/2024 9:15:16 AM] UniGetUI is up to date
[9/13/2024 9:21:11 AM] Found 0 available updates from Chocolatey
[9/13/2024 9:21:11 AM] Found 0 available updates from Pip
[9/13/2024 9:21:14 AM] Found 0 available updates from PowerShell
[9/13/2024 9:21:15 AM] Found 6 available updates from Winget
[9/13/2024 9:21:16 AM] Found 0 available updates from PowerShell7
[9/13/2024 10:15:16 AM] UniGetUI is up to date
[9/13/2024 10:21:26 AM] Found 0 available updates from Chocolatey
[9/13/2024 10:21:27 AM] Found 0 available updates from Pip
[9/13/2024 10:21:32 AM] Found 6 available updates from Winget
[9/13/2024 10:21:34 AM] Found 0 available updates from PowerShell7
[9/13/2024 10:21:35 AM] Found 0 available updates from PowerShell
[9/13/2024 11:18:38 AM] UniGetUI is up to date
[9/13/2024 11:25:03 AM] Found 0 available updates from Pip
[9/13/2024 11:25:05 AM] Found 0 available updates from Chocolatey
[9/13/2024 11:25:06 AM] Found 6 available updates from Winget
[9/13/2024 11:25:08 AM] Found 0 available updates from PowerShell7
[9/13/2024 11:25:08 AM] Found 0 available updates from PowerShell
[9/13/2024 12:18:39 PM] UniGetUI is up to date
[9/13/2024 12:25:15 PM] Found 0 available updates from Chocolatey
[9/13/2024 12:25:16 PM] Found 0 available updates from Pip
[9/13/2024 12:25:17 PM] Found 6 available updates from Winget
[9/13/2024 12:25:18 PM] Found 0 available updates from PowerShell
[9/13/2024 12:25:20 PM] Found 0 available updates from PowerShell7
[9/13/2024 1:15:11 PM] Found 1 installed packages from Pip
[9/13/2024 1:15:15 PM] Found 12 installed packages from Chocolatey
[9/13/2024 1:15:16 PM] Found 0 installed packages from PowerShell7
[9/13/2024 1:15:16 PM] Found 1 installed packages from PowerShell
[9/13/2024 1:15:20 PM] Found 179 installed packages from Winget
[9/13/2024 1:15:20 PM] Adding missing package ARP\Machine\X64\{A7AB73A3-CB10-4AA5-9D38-6AEFFBDE4C91} to installed packages list
[9/13/2024 1:15:20 PM] Adding missing package Microsoft.Teams to installed packages list
[9/13/2024 1:15:20 PM] Adding missing package MSIX\Microsoft.Winget.Source_2024.913.1700.55_neutral__8wekyb3d8bbwe to installed packages list
[9/13/2024 1:15:20 PM] Adding missing package MSIX\MicrosoftWindows.CrossDevice_1.24081.57.0_x64__cw5n1h2txyewy to installed packages list
[9/13/2024 1:18:39 PM] UniGetUI is up to date
```

### Package Managers Logs

```text
Manager WinGet with version:
System WinGet CLI Version: v1.8.1911
Microsoft.WinGet.Client PSModule version: ""
Using Native WinGet helper (COM Api)

——————————————————————————————————————————


Logged native task on manager Winget. Task type is ListSources
Process start time: 9/12/2024 8:19:45 AM
Process end time:   9/12/2024 8:19:45 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 8:19:52 AM
Process end time:   9/12/2024 8:20:05 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListInstalledPackages
Process start time: 9/12/2024 8:19:53 AM
Process end time:   9/12/2024 8:20:34 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListSources
Process start time: 9/12/2024 8:19:57 AM
Process end time:   9/12/2024 8:19:59 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 8:20:05 AM
Process end time:   9/12/2024 8:20:32 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListInstalledPackages
Process start time: 9/12/2024 8:20:55 AM
Process end time:   9/12/2024 8:21:02 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 9:20:53 AM
Process end time:   9/12/2024 9:20:54 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 9:20:54 AM
Process end time:   9/12/2024 9:21:02 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 10:28:01 AM
Process end time:   9/12/2024 10:28:03 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 10:28:04 AM
Process end time:   9/12/2024 10:28:10 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 11:28:10 AM
Process end time:   9/12/2024 11:28:13 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 11:28:13 AM
Process end time:   9/12/2024 11:28:20 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 12:28:20 PM
Process end time:   9/12/2024 12:28:25 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 12:28:25 PM
Process end time:   9/12/2024 12:28:34 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 1:28:35 PM
Process end time:   9/12/2024 1:28:40 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 1:28:40 PM
Process end time:   9/12/2024 1:28:47 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 2:28:49 PM
Process end time:   9/12/2024 2:28:50 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 2:28:50 PM
Process end time:   9/12/2024 2:29:02 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 3:29:09 PM
Process end time:   9/12/2024 3:29:10 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 3:29:10 PM
Process end time:   9/12/2024 3:29:18 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 4:29:23 PM
Process end time:   9/12/2024 4:29:25 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 4:29:25 PM
Process end time:   9/12/2024 4:29:33 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListInstalledPackages
Process start time: 9/12/2024 4:29:49 PM
Process end time:   9/12/2024 4:29:53 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 6:45:48 PM
Process end time:   9/12/2024 6:45:49 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 6:45:49 PM
Process end time:   9/12/2024 6:45:58 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 7:46:00 PM
Process end time:   9/12/2024 7:46:04 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 7:46:04 PM
Process end time:   9/12/2024 7:46:12 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 8:46:13 PM
Process end time:   9/12/2024 8:46:15 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 8:46:15 PM
Process end time:   9/12/2024 8:46:21 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 9:46:21 PM
Process end time:   9/12/2024 9:46:22 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 9:46:23 PM
Process end time:   9/12/2024 9:46:28 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 10:46:29 PM
Process end time:   9/12/2024 10:46:32 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 10:46:32 PM
Process end time:   9/12/2024 10:46:38 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/12/2024 11:46:38 PM
Process end time:   9/12/2024 11:46:39 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/12/2024 11:46:39 PM
Process end time:   9/12/2024 11:46:49 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 12:46:49 AM
Process end time:   9/13/2024 12:46:52 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 12:46:52 AM
Process end time:   9/13/2024 12:46:58 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 1:46:59 AM
Process end time:   9/13/2024 1:47:01 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 1:47:01 AM
Process end time:   9/13/2024 1:47:09 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 2:47:11 AM
Process end time:   9/13/2024 2:47:12 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 2:47:12 AM
Process end time:   9/13/2024 2:47:21 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 3:47:22 AM
Process end time:   9/13/2024 3:47:26 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 3:47:26 AM
Process end time:   9/13/2024 3:47:34 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 4:47:34 AM
Process end time:   9/13/2024 4:47:38 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 4:47:38 AM
Process end time:   9/13/2024 4:47:48 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 5:47:50 AM
Process end time:   9/13/2024 5:47:52 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 5:47:52 AM
Process end time:   9/13/2024 5:47:59 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 6:47:59 AM
Process end time:   9/13/2024 6:48:02 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 6:48:02 AM
Process end time:   9/13/2024 6:48:09 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 8:20:33 AM
Process end time:   9/13/2024 8:20:42 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 8:20:42 AM
Process end time:   9/13/2024 8:20:59 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 9:21:03 AM
Process end time:   9/13/2024 9:21:06 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 9:21:06 AM
Process end time:   9/13/2024 9:21:15 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 10:21:16 AM
Process end time:   9/13/2024 10:21:22 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 10:21:22 AM
Process end time:   9/13/2024 10:21:32 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 11:24:56 AM
Process end time:   9/13/2024 11:25:01 AM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 11:25:01 AM
Process end time:   9/13/2024 11:25:06 AM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\user\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 9/13/2024 12:25:08 PM
Process end time:   9/13/2024 12:25:09 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 9/13/2024 12:25:09 PM
Process end time:   9/13/2024 12:25:17 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListInstalledPackages
Process start time: 9/13/2024 1:15:10 PM
Process end time:   9/13/2024 1:15:20 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————
```

### Relevant information

I have the following packages in my ignore list:

* `Hashicrop.Terraform`: All versions
* `SomePythonThings.WingetUIStore.Pre-release`: 3.1.0-beta1

**Comment by marticliment** [comment]
Will investigate. Perhaps the issue was not 100% fixed

**Comment by moorereason** [comment]
It happened again this afternoon.  I can confirm that maximizing the window is also a workaround.  I'm assuming at this point that any resizing of the window or package listing pane is a workaround.

More:  I noticed the **VLC media player** update via winget has been reporting as successfully installed but then shows back up again later as not installed, so I unchecked that package on my most recent package updates.  When I saw the issue today, only the **VLC media player** update was available and unchecked.  I don't know if that's relevant but wanted to share just in case it is.

**Comment by arogl** [comment]
I'll comment here, but happy to create new issue.

I have not tested maximising the window. This all worked previously under WinGetUI 2.x. I thought it may have been just a beta issue, but has been all in 3.x releases.

When starting UniGetUI, only the `winget` updates are shown:

Output from a pip check:

```
pip list --outdated --format=json |  jq -r "map(.name)| .[]"
bandcamp-downloader
jaraco.functools
numpy
virtualenv
```

I have marked `numpy` as not to be updated in UniGetUI.

UniGetUI on startup:
![uniget-start](https://github.com/user-attachments/assets/891d4566-917f-40f5-a05e-e883674a742e)

<details>

<summary>UniGetUI log at startup</summary>

```
[28/09/2024 2:15:14 PM]    __  __      _ ______     __  __  ______
                          / / / /___  (_) ____/__  / /_/ / / /  _/
                         / / / / __ \/ / / __/ _ \/ __/ / / // /
                        / /_/ / / / / / /_/ /  __/ /_/ /_/ // /
                        \____/_/ /_/_/\____/\___/\__/\____/___/
                            Welcome to UniGetUI Version 3.1.2-beta3
[28/09/2024 2:15:14 PM]
[28/09/2024 2:15:14 PM] Version Code:  3.1193
[28/09/2024 2:15:14 PM] Encoding Code Page set to 65001
[28/09/2024 2:15:15 PM] Using bundled GSudo at C:\Program Files\UniGetUI\Assets\Utilities\gsudo.exe
[28/09/2024 2:15:16 PM] Loaded language locale: en
[28/09/2024 2:15:17 PM] Lang files were updated successfully from GitHub
[28/09/2024 2:15:18 PM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                        █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                        █ Name: .NET Tool
                        █ Enabled: False
                        █ THE MANAGER IS DISABLED
                        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[28/09/2024 2:15:18 PM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                        █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                        █ Name: PowerShell
                        █ Enabled: False
                        █ THE MANAGER IS DISABLED
                        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[28/09/2024 2:15:18 PM] Downloaded new icons and screenshots successfully!
[28/09/2024 2:15:19 PM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                        █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                        █ Name: Winget
                        █ Enabled: True
                        █ Found: True
                        █ Fancy exe name: winget.exe
                        █ Executable path: C:\Users\arogl\AppData\Local\Microsoft\WindowsApps\winget.exe
                        █ Call arguments: 
                        █ Version: 
                        █   System WinGet CLI Version: v1.9.2411-preview
                        █   Using Native WinGet helper (COM Api)
                        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[28/09/2024 2:15:20 PM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                        █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                        █ Name: Cargo
                        █ Enabled: True
                        █ Found: True
                        █ Fancy exe name: cargo.exe
                        █ Executable path: C:\Users\arogl\.cargo\bin\cargo.exe
                        █ Call arguments: 
                        █ Version: 
                        █   cargo 1.80.1 (376290515 2024-07-16)
                        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[28/09/2024 2:15:20 PM] Command scoop was not found on the system
[28/09/2024 2:15:20 PM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                        █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                        █ Name: Scoop
                        █ Enabled: False
                        █ THE MANAGER IS DISABLED
                        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[28/09/2024 2:15:20 PM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                        █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                        █ Name: Pip
                        █ Enabled: True
                        █ Found: True
                        █ Fancy exe name: pip
                        █ Executable path: C:\Users\arogl\AppData\Local\Programs\Python\Python312\python.exe
                        █ Call arguments:  -m pip
                        █ Version: 
                        █   pip 24.2 from C:\Users\arogl\AppData\Local\Programs\Python\Python312\Lib\site-packages\pip (python 3.12)
                        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[28/09/2024 2:15:21 PM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                        █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                        █ Name: Npm
                        █ Enabled: False
                        █ THE MANAGER IS DISABLED
                        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[28/09/2024 2:15:22 PM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                        █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                        █ Name: Chocolatey
                        █ Enabled: True
                        █ Found: True
                        █ Fancy exe name: choco.exe
                        █ Executable path: C:\Users\arogl\AppData\Local\UniGetUI\Chocolatey\choco.exe
                        █ Call arguments: 
                        █ Version: 
                        █   2.2.2
                        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[28/09/2024 2:15:25 PM] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                        █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                        █ Name: PowerShell7
                        █ Enabled: True
                        █ Found: True
                        █ Fancy exe name: pwsh.exe
                        █ Executable path: C:\Program Files\PowerShell\7\pwsh.exe
                        █ Call arguments:  -NoProfile -Command
                        █ Version: 
                        █   PowerShell 7.4.5
                        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[28/09/2024 2:15:25 PM] LoadComponentsAsync finished executing. All managers loaded. Proceeding to interface.
[28/09/2024 2:15:27 PM] Found 7 installed packages from Cargo
[28/09/2024 2:15:29 PM] Found 5 installed packages from Chocolatey
[28/09/2024 2:15:29 PM] Dependency cargo-update for manager Cargo is present
[28/09/2024 2:15:30 PM] Ignored updates database was never loaded, so it is going to be loaded now
[28/09/2024 2:15:31 PM] Found 0 available updates from Cargo
[28/09/2024 2:15:31 PM] Found 2 installed packages from PowerShell7
[28/09/2024 2:15:35 PM] Found 0 available updates from Chocolatey
[28/09/2024 2:15:35 PM] Found 313 installed packages from Pip
[28/09/2024 2:15:39 PM] Found 0 available updates from PowerShell7
[28/09/2024 2:16:13 PM] Found 5 available updates from Winget
[28/09/2024 2:16:13 PM] Found 206 installed packages from Winget
[28/09/2024 2:16:14 PM] Backup saved to C:\Users\arogl\OneDrive\Documents\UniGetUI\MSI-GE62 installed packages 2024-09-28 14-16-14.ubundle
[28/09/2024 2:16:25 PM] Error finding updates on manager Pip
[28/09/2024 2:16:25 PM] System.TimeoutException: The operation has timed out.
                           at UniGetUI.PackageEngine.ManagerClasses.Manager.PackageManager.GetAvailableUpdates() in C:\SomePrograms\WingetUI-Store\src\UniGetUI.PackageEngine.PackageManagerClasses\Manager\PackageManager.cs:line 199
```
</details>

<details>

<summary>UniGetUI Packet Manager log at startup</summary>

```
Manager WinGet with version:
System WinGet CLI Version: v1.9.2411-preview
Using Native WinGet helper (COM Api)

——————————————————————————————————————————


Logged native task on manager Winget. Task type is ListSources
Process start time: 28/09/2024 2:15:19 PM
Process end time:   28/09/2024 2:15:19 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListInstalledPackages
Process start time: 28/09/2024 2:15:25 PM
Process end time:   28/09/2024 2:16:13 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\arogl\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 28/09/2024 2:15:25 PM
Process end time:   28/09/2024 2:15:33 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListSources
Process start time: 28/09/2024 2:15:27 PM
Process end time:   28/09/2024 2:15:27 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 28/09/2024 2:15:33 PM
Process end time:   28/09/2024 2:16:13 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————
```

</details>

After clicking the refresh button:

![uniget-update](https://github.com/user-attachments/assets/85b4ba91-5d00-461b-9353-599e87ea1a4d)

Updated log files, just showing the extra lines added in the session.

<details>

<summary>UniGetUI log</summary>

```
[28/09/2024 2:18:43 PM] Found 0 available updates from Cargo
[28/09/2024 2:18:46 PM] Found 0 available updates from Chocolatey
[28/09/2024 2:18:47 PM] Found 5 available updates from Winget
[28/09/2024 2:18:51 PM] Found 0 available updates from PowerShell7
[28/09/2024 2:19:08 PM] Found 5 available updates from Pip
```
</details>

<details>

<summary>UniGetUI Packet Manager logs</summary>

```log

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\arogl\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 28/09/2024 2:18:43 PM
Process end time:   28/09/2024 2:18:43 PM

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListUpdates
Process start time: 28/09/2024 2:18:43 PM
Process end time:   28/09/2024 2:18:47 PM

-- Task information
 ...

The task reported success

——————————————————————————————————————————
```

</details>

**Comment by arogl** [comment]
The same issue is still occuring in version 3.1.3.

**Comment by arogl** [comment]
> The same issue is still occuring in version 3.1.3.

Disabling the 1-minute timeout for package related operations has allowed `pip` to complete

**Comment by moorereason** [comment]
> Disabling the 1-minute timeout for package related operations has allowed `pip` to complete

Does enabling this setting resolve the rendering problem I described or are you talking about a different issue?  I'm skeptical that this setting would resolve my issue, which I'm still seeing in 3.1.5.

**Comment by marticliment** [comment]
No, this issue here is a completely different thing

**Comment by marticliment** [comment]
Is this issue still happening on 3.2.0?

**Comment by MikaelKW** [comment]
Not seen this issue for quite some versions now.

**Comment by marticliment** [comment]
I think https://github.com/marticliment/UniGetUI/pull/3597 fixed it

**Comment by moorereason** [comment]
I haven't seen this issue lately either.  v3.2.0 was released on May 7th -- that sounds about right for when the issue probably went away.  🎉
